test/messages.js

```diff
@@ -244,9 +244,11 @@ tape('double close', function (t) {
 })
 
 tape('properly close if destroy called with a open request', function (t) {
+  t.plan(2)
 
   var a = ps({
     request: function (value, cb) {
+      t.equals(value, 7)
       // never calls cb
```

Comment:
Shouldn't this comment then be removed?

Reply:
The way I understood it, the `cb` on line 250 should never be called, which is true inside this block, but `request` gets called with (value,cb) as args.

It's good to review with care, though, because after all this sits in the core of everything